extensions/cesium-renderer/src/decorations/CesiumCoordinateConverter.ts

```diff
@@ -0,0 +1,279 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import { Cartographic, EcefLocation, ViewDefinition3dProps } from "@itwin/core-common";
+import { Point3d, Range3d, Vector3d, XYAndZ, YawPitchRollAngles } from "@itwin/core-geometry";
+import { IModelConnection } from "@itwin/core-frontend";
+import { Cartesian3, Cartographic as CesiumCartographic } from "cesium";
+
+/**
+ * Interface for CesiumJS frustum parameters
+ */
+interface CesiumFrustum {
+  near: number;
+  far: number;
+  fov?: number;
+  width?: number;
+}
+
+/**
+ * Interface for CesiumJS camera parameters
+ */
+interface CesiumCamera {
+  position: Point3d;
+  direction: Vector3d;
+  up: Vector3d;
+  frustum: CesiumFrustum;
+}
+
+/**
+ * Professional coordinate conversion utilities between iTwin.js and CesiumJS
+ * Handles spatial coordinate transformations, camera conversions, and ECEF mappings
+ */
+export class CesiumCoordinateConverter {
+  private _iModel: IModelConnection;
+  private _ecefLocation?: EcefLocation;
+
+  constructor(iModel: IModelConnection) {
+    this._iModel = iModel;
+    this._ecefLocation = iModel.ecefLocation;
+    
+  }
+
+  /**
+   * Convert iTwin.js spatial coordinates to CesiumJS Cartesian3 (ECEF coordinates)
+   * Uses iTwin.js built-in spatialToEcef method from IModel.ts:636
```

Comment:
Nit: this and some of the other docs/comments in `cesiumCartesian3ToSpatial` and `spatialToGeographic` feel unnecessary and add clutter to these functions. For example, there's no need to document what `spatialToEcef` does or its file/line number when someone can just check the import statement.